source/SyntheticTopology/SierpinskiAxioms.lagda

```diff
@@ -0,0 +1,124 @@
+---
+title:        Axioms about the Sierpinski object
+authors:      [Martin Trucchi , Ayberk Tosun]
```

Comment:
Can you wrap names in quotes here? It might work without the quotes, but I’m not sure.

Reply:
Simple or double quotes ?

Reply:
Done!